src/modules/launcher/Plugins/Microsoft.Plugin.Uri.UnitTests/UriHelper/ExtendedUriParserTests.cs

```diff
@@ -56,6 +56,10 @@ public class ExtendedUriParserTests
         [DataRow("tel:411", true, "tel:411")]
         [DataRow("ftp://example.com", true, "ftp://example.com/")]
         [DataRow("example.com:443", true, "example.com:443")]
+        [DataRow("mailto:", true, "mailto:")]
+        [DataRow("mailto:/", true, "mailto:/")]
+        [DataRow("ms-settings:", true, "ms-settings:")]
+        [DataRow("ms-settings:/", true, "ms-settings:/")]
```

Comment:
It is `microsoft-edge:http://goole.com` and `microsoft-edge:goole.com`.

Can you plz try it.

Reply:
It won't work cause because this is not a scheme only uri and this part will handle it: 
https://github.com/microsoft/PowerToys/blob/6679d047342ba788729115fac4849fd3f2e7d18e/src/modules/launcher/Plugins/Microsoft.Plugin.Uri/UriHelper/ExtendedUriParser.cs#L44-L61
![image](https://user-images.githubusercontent.com/11349917/139568058-46888434-2c52-42db-b45d-078148cdb4b4.png)
I'll go ahead and add support for this. :)

Reply:
I've added the support for these and updated the title and content of this PR. Please re-review the changes. @htcfreek, @davidegiacometti